packages/shorebird_cli/lib/src/commands/account/create_account_command.dart

```diff
@@ -38,7 +38,7 @@ class CreateAccountCommand extends ShorebirdCommand with ShorebirdConfigMixin {
       // status.
       logger.info('''
 You already have an account, ${error.user.displayName}!
-To upgrade, run ${lightCyan.wrap('shorebird account upgrade')}.
+To login, run ${lightCyan.wrap('shorebird login')}.
```

Comment:
Is it possible to just log the user in at this point?

Reply:
We'd have to force them through the auth flow so I felt it'd be cleaner to direct them to the login command to keep things simple.